hackathons.geojson

```diff
@@ -71,11 +71,33 @@
             },
             "type": "Feature"
         },
+
+
+
+	{
+	    "geometry": {
+		"coordinates": [
+		    13.33,
+		    52.44
```

Comment:
Please lookup exact coordinates for `Mainzer Str. 16, 10247 Berlin` using Google Maps.